### DISCUSSION TITLE
Does the mod spawn any zombies or are they just attracted by the sound of the helicopters?

### DISCUSSION THREAD

**Discussion by LukaKeres** [TEHE-Studios/ExpandedHelicopterEvents#322] [Q&A] [answered]
i woke up to huge swarm of zombies on my backyard

**Comment by Chuckleberry-Finn** [comment, marked as answer]
The only time zombies are spawned in is during a crash which may spawn 1-2 zombies in air force uniforms.

In super weird Helis the tax man event can spawn a dozen.

Other than that all the zombies you see are lured in from sound.
